internal/gengapic/genrest.go

```diff
@@ -112,7 +159,59 @@ func (g *generator) genRESTMethod(servName string, serv *descriptor.ServiceDescr
 	}
 }
 
+func (g *generator) emptyUnaryRESTCall(servName string, m *descriptor.MethodDescriptorProto) error {
+	info, err := getHttpInfo(m)
+	if err != nil || info == nil {
+		return err
+	}
+
+	inType := g.descInfo.Type[*m.InputType]
+	inSpec, err := g.descInfo.ImportSpec(inType)
+	if err != nil {
+		return err
+	}
+
+	p := g.printf
+	lowcaseServName := lowcaseRestClientName(servName)
+	p("func (c *%s) %s(ctx context.Context, req *%s.%s, opts ...gax.CallOption) error {",
+		lowcaseServName, m.GetName(), inSpec.Name, inType.GetName())
+
+	// TODO(dovs): handle cancellation, metadata, osv.
+	// TODO(dovs): handle http headers
+	// TODO(dovs): handle deadlines?
+	// TODO(dovs): handle call options
+	p("// The default (false) for the other options are fine.")
+	p("m := protojson.MarshalOptions{AllowPartial: true, Multiline: true, EmitUnpopulated: true}")
```

Comment:
I was leanig towards multiline because it's easier to eyeball parse, but I have no strong feelings on the matter. If you think the extra bytes aren't worth it I'll change this.

Reply:
Removing multiline.